resources/inputmaps/apple-media-keys.json

```diff
@@ -3,7 +3,9 @@
   "idmatcher": "AppleMediaKeys.*",
   "mapping":
   {
-    "KEY_PLAY": "play_pause",
+    "KEY_PLAY": "play",
+    "KEY_PAUSE": "pause",
+    "KEY_PLAY_PAUSE": "play_pause",
```

Comment:
Probably not, which is why I'm not overly worried.

Maybe it could be an entry in the changelog.